src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const replaceStr = str.replace(/\W/g, '').toLowerCase();
```

Comment:
```suggestion
  const replacedString = str.replace(/\W/g, '').toLowerCase();
```

переменные над которыми произвелись операции пишутся через `ed` - результат действия
 `replace` => `replaced` `reverse` => `reversed`